index.js

```diff
@@ -39,6 +39,7 @@ function ZongJi(dsn, options) {
 
     binlogDsn = dsn;
   }
+  this.connectionConfig = cloneObjectSimple(binlogDsn);
```

Comment:
Why create a clone instead of reading from `zongji.connection.config`?

Reply:
I missed that.  I'll amend the commit.